.travis.yml

```diff
@@ -27,6 +27,7 @@ env:
   - DB=postgres
 
 install:
+  - pip install --upgrade pip # Prevent old pip incompatibilities.
```

Comment:
I'd like to see a more explicit comment re: why we're doing this -- I don't like workarounds that don't clearly explain why they're there. Otherwise I'm happy though.